build.gradle

```diff
@@ -86,6 +86,7 @@ dependencies {
     deobfCompile "curse.maven:mixinbooter-419286:3575119:"
     compileOnly files('deps/GalacticraftCore-1.12.2-4.0.2.280-deobf.jar')
     compileOnly files('deps/MicdoodleCore-1.12.2-4.0.2.280-deobf.jar')
+    compileOnly files('deps/WitcheryResurrected-1.12.2-0.5.2.4-forge-all.jar')
```

Comment:
FYI, I found a trick to avoid needing to include their JAR in the repository, so I'll push that before merging. I'd like to avoid doing what I did with Galacticraft unless absolutely necessary, especially for mods that are actively being maintained.

Reply:
I'm not very familiar with Gradle, do you have any tips? This mod is not on CurseForge but does however have a Website.
Link to current `.jar`
https://witchery.msrandom.net/download/WitcheryResurrected0.5.2Hotfix4/WitcheryResurrected-1.12.2-0.5.2.4-forge-all.jar